src/services/API/Plio.js

```diff
@@ -85,14 +90,30 @@ export default {
 
   duplicatePlio(plioId) {
     // create a clone of plioId plio
-    return apiClient().post(pliosEndpoint + plioId + "/duplicate");
+    return apiClient().post(pliosEndpoint + plioId + duplicatePlioEndpoint);
   },
 
   async getUniqueUsersCount(plioId) {
     // get the count of unique users who watched the given plio
-    var response = await apiClient().get(
-      pliosEndpoint + plioId + "/unique_users"
-    );
-    return response.data;
+    // refer to this example: https://cube.dev/blog/vue-dashboard-tutorial-using-cubejs/
+
+    // this is the query in the CubeJS Data Schema which is automatically
+    // converted to SQL by CubeJS: https://cube.dev/docs/getting-started-cubejs-schema
+    var uniqueUsersQuery = {
+      measures: ["Session.uniqueUsers"],
+      filters: [
+        {
+          member: "Plio.uuid",
+          operator: "equals",
+          values: [plioId],
+        },
+      ],
+    };
```

Comment:
I think it's best to encapsulate this query and all the queries that we will use soon in a separate file and import them here. Just like we've done for endpoints and API calls.
A query directory can have different files like Users, Session, Plios etc and each file will have the related query in a variable or const